protocol/src/main/protobuf/Transactor.proto

```diff
@@ -14,14 +14,34 @@ message UpdateRequest {
     bytes chainCellCbor = 1;
 }
 
+message JournalStreamRequest {
+    MultihashReference lastJournalBlock = 1;
+}
+
+message ChainUpdateDescription {
+    MultihashReference canonicalRef = 1;
+    MultihashReference chainHeadRef = 2;
+}
+
+message JournalEvent {
+    oneof event {
+        MultihashReference canonicalInserted = 2;
+        ChainUpdateDescription chainUpdated = 3;
+        MultihashReference journalBlockPublished = 4;
+    }
+}
+
 service TransactorService {
     rpc InsertCanonical(InsertRequest) returns
         (MultihashReference);
 
     rpc UpdateChain(UpdateRequest) returns
-        (MultihashReference);
+        (ChainUpdateDescription);
```

Comment:
yeah, this name kind of sucks.  I changed the return type to more closely match the ChainEntry, where you have a ref to the canonical and the cell instead of just the cell.  I figure it could be useful in a client, because you might only care about updates to a subset of all canonicals, so having the canonical ref could be handy.  `UpdateChainResult` sounds good tho